tasks/packages.yml

```diff
@@ -22,7 +22,7 @@
       {% if item.repos is defined %}{{ item.repos }}{% endif %}
   register: r_install_package
   changed_when: "r_install_package.stdout_lines[-1] is defined and r_install_package.stdout_lines[-1] == 'changed'"
-  with_items: "{{ r_packages }}"
+  with_items: "{{ ( [{ 'name' : 'remotes' }] if r_packages_install_remotes == true else [] ) + r_packages }}"
```

Comment:
Do you think it's possible to do this check based on the `type` property of `r_packages`?

Reply:
Oh sorry, this might be ambiguous and you might decide I should recode this. All it's doing is prepending the `remotes` package from CRAN to the installation list if the user wants to have it. I did this way to avoid adding another task, but it results in some messiness with the packages list. I can fix this and split into another task if you think it's better stylistically.